src/tcutility/results/adf.py

```diff
@@ -62,6 +69,57 @@ def get_calc_settings(info: Result) -> Result:
     return ret
 
 
+# ------------------------------------------------------------- #
+# ------------------------ Properties ------------------------- #
+# ------------------------------------------------------------- #
+
+
+# ----------------------- VDD charges ------------------------- #
+
+
+def _read_vdd_charges(kf_reader: KFReader) -> arrays.Array1D[np.float64]:
```

Comment:
Is there a reason to return an array instead of a list?

Reply:
Numpy arrays are more favored when dealing with floating numbers because these are 
- faster (better memory allocation)
- use less memory (better memory allocation)
- have useful [class methods](https://numpy.org/doc/stable/reference/generated/numpy.ndarray.html) 

There is not much benefit for this specific use in terms of speed and memory use because the arrays are quite small (size of number of atoms times number of irreps). But, I think it's good practice

Reply:
I would recommend using numpy arrays everywhere, but we can do this in another update